Yoast/wordpress-seo
# import yoast components without paths

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Replacing all yoast-components imports with imports that do not require a path.

## Test instructions

This PR can be tested by following these steps:

* use [this branch of yoast-components](https://github.com/Yoast/yoast-components/pull/687/) to test this PR with (`yarn link` it).
* Install the plugin by checking out this branch and running `yarn && composer install && grunt`
* Verify there are no errors in the console.

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo-premium/issues/1901
depends: https://github.com/Yoast/yoast-components/pull/687/
